cubedash/_stac.py

```diff
@@ -1370,7 +1363,7 @@ def arrivals_items():
     """
     limit = request.args.get("limit", default=get_default_limit(), type=int)
     offset = request.args.get("_o", default=0, type=int)
-    check_page_limit(limit)
+    limit = check_page_limit(limit)
```

Comment:
If you wrap `check_page_limit()` around the RHS of the previous line that defines `limit`, it will be impossible for future changes to use the wrong limit value.

Same comment for the other places with this change.